SalesforceSDK/Salesforce.SDK.App/Hybrid/HybridMainPage.cs

```diff
@@ -183,11 +183,12 @@ protected void OnResumeLoggedInNotLoaded()
                     // Offline
                 else
                 {
+                    // TODO: FIXME
                     // Has cached version
-                    if (false /* FIXME */)
-                    {
-                        Log("Success:Loading cached version of remote application because offline");
-                    }
+                    //if (false)
```

Comment:
This is for caching and we don't have that implemented yet. @kchitalia will be working on fixing up HybridMainPage and cleaning up as needed. I just left this TODO here so we don't forget about this.